File: mobile_app/app_ios/Objects/src/B4iProject/LS_frminfoempresa.m

#import "iCore.h"
@interface LS_frminfoempresa:NSObject
@end

@implementation LS_frminfoempresa

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[views get:@"panelinfoback"].Left = (int)(0.0);
[views get:@"panelinfoback"].Width = (int)((100 / 100.0 * width) - (0.0));
[views get:@"panelinfoback"].Top = (int)(0.0);
[views get:@"panelinfoback"].Height = (int)((100 / 100.0 * height) - (0.0));
[views get:@"panelinfo"].Left = (int)((7 / 100.0 * width));
[views get:@"panelinfo"].Height = (int)((51 / 100.0 * height));
[views get:@"panelinfo"].Top = (int)((50 / 100.0 * height) - ([views get:@"panelinfo"].Height / 2));
[views get:@"panelinfo"].Width = (int)((100 / 100.0 * width)-([views get:@"panelinfo"].Left)*2.0);
[views get:@"lblinfoempresa"].Top = (int)((3 / 100.0 * height));
[views get:@"lblinfoempresa"].Left = (int)((5 / 100.0 * width));
[views get:@"lblinfoempresa"].Height = (int)((5 / 100.0 * height));
[views get:@"lblinfoempresa"].Width = (int)(([views get:@"panelinfo"].Width)-([views get:@"lblinfoempresa"].Left));
[views get:@"label1"].Left = (int)(([views get:@"lblinfoempresa"].Left));
[views get:@"label1"].Height = (int)((5 / 100.0 * height));
[views get:@"label1"].Width = (int)((35 / 100.0 * width));
[views get:@"label1"].Top = (int)(([views get:@"lblinfoempresa"].Top + [views get:@"lblinfoempresa"].Height)+(1 / 100.0 * height));
[views get:@"lblinfonegocio"].Top = (int)(([views get:@"label1"].Top));
[views get:@"lblinfonegocio"].Left = (int)(([views get:@"label1"].Left + [views get:@"label1"].Width));
[views get:@"lblinfonegocio"].Height = (int)(([views get:@"label1"].Height));
[views get:@"lblinfonegocio"].Width = (int)(([views get:@"panelinfo"].Width)-([views get:@"lblinfonegocio"].Left));
[views get:@"label2"].Left = (int)(([views get:@"label1"].Left));
[views get:@"label2"].Width = (int)(([views get:@"label1"].Width));
[views get:@"label2"].Height = (int)(([views get:@"label1"].Height));
[views get:@"label2"].Top = (int)(([views get:@"label1"].Top + [views get:@"label1"].Height)+(1 / 100.0 * height));
[views get:@"lblpedidominimo"].Top = (int)(([views get:@"label2"].Top));
[views get:@"lblpedidominimo"].Left = (int)(([views get:@"label1"].Left + [views get:@"label1"].Width));
[views get:@"lblpedidominimo"].Height = (int)(([views get:@"label1"].Height));
[views get:@"lblpedidominimo"].Width = (int)(([views get:@"lblinfonegocio"].Width));
[views get:@"label3"].Left = (int)(([views get:@"label1"].Left));
[views get:@"label3"].Width = (int)(([views get:@"label1"].Width)/1.5);
[views get:@"label3"].Height = (int)(([views get:@"label1"].Height));
[views get:@"label3"].Top = (int)(([views get:@"label2"].Top + [views get:@"label2"].Height)+(1 / 100.0 * height));
[views get:@"lbltelefono"].Top = (int)(([views get:@"label3"].Top));
//BA.debugLineNum = 43;BA.debugLine="lblTelefono.Left = Label3.Right"[frmInfoEmpresa/General script]
[views get:@"lbltelefono"].Left = (int)(([views get:@"label3"].Left + [views get:@"label3"].Width));
//BA.debugLineNum = 44;BA.debugLine="lblTelefono.Height = Label1.Height"[frmInfoEmpresa/General script]
[views get:@"lbltelefono"].Height = (int)(([views get:@"label1"].Height));
//BA.debugLineNum = 45;BA.debugLine="lblTelefono.Width = lblInfoNegocio.Width"[frmInfoEmpresa/General script]
[views get:@"lbltelefono"].Width = (int)(([views get:@"lblinfonegocio"].Width));
//BA.debugLineNum = 47;BA.debugLine="Label4.Left = Label1.Left"[frmInfoEmpresa/General script]
[views get:@"label4"].Left = (int)(([views get:@"label1"].Left));
//BA.debugLineNum = 48;BA.debugLine="Label4.Width = Label1.Width"[frmInfoEmpresa/General script]
[views get:@"label4"].Width = (int)(([views get:@"label1"].Width));
//BA.debugLineNum = 49;BA.debugLine="Label4.Height = Label1.Height"[frmInfoEmpresa/General script]
[views get:@"label4"].Height = (int)(([views get:@"label1"].Height));
//BA.debugLineNum = 50;BA.debugLine="Label4.Top = Label3.Bottom + 1%y"[frmInfoEmpresa/General script]
[views get:@"label4"].Top = (int)(([views get:@"label3"].Top + [views get:@"label3"].Height)+(1 / 100.0 * height));
//BA.debugLineNum = 52;BA.debugLine="lblDireccion.Top = Label4.Bottom"[frmInfoEmpresa/General script]
[views get:@"lbldireccion"].Top = (int)(([views get:@"label4"].Top + [views get:@"label4"].Height));
//BA.debugLineNum = 53;BA.debugLine="lblDireccion.Left = Label4.Left"[frmInfoEmpresa/General script]
[views get:@"lbldireccion"].Left = (int)(([views get:@"label4"].Left));
//BA.debugLineNum = 54;BA.debugLine="lblDireccion.Height = 10%y"[frmInfoEmpresa/General script]
[views get:@"lbldireccion"].Height = (int)((10 / 100.0 * height));
//BA.debugLineNum = 55;BA.debugLine="lblDireccion.Width = PanelInfo.Width - lblDireccion.Left *2"[frmInfoEmpresa/General script]
[views get:@"lbldireccion"].Width = (int)(([views get:@"panelinfo"].Width)-([views get:@"lbldireccion"].Left)*2.0);
//BA.debugLineNum = 57;BA.debugLine="btnInfoCerrar.Width = 25%x"[frmInfoEmpresa/General script]
[views get:@"btninfocerrar"].Width = (int)((25 / 100.0 * width));
//BA.debugLineNum = 58;BA.debugLine="btnInfoCerrar.Height = 7%y"[frmInfoEmpresa/General script]
[views get:@"btninfocerrar"].Height = (int)((7 / 100.0 * height));
//BA.debugLineNum = 59;BA.debugLine="btnInfoCerrar.Right = PanelInfo.Width - 2%x"[frmInfoEmpresa/General script]
[views get:@"btninfocerrar"].Left = (int)(([views get:@"panelinfo"].Width)-(2 / 100.0 * width) - ([views get:@"btninfocerrar"].Width));
//BA.debugLineNum = 60;BA.debugLine="btnInfoCerrar.Bottom = PanelInfo.Height -1%y"[frmInfoEmpresa/General script]
[views get:@"btninfocerrar"].Top = (int)(([views get:@"panelinfo"].Height)-(1 / 100.0 * height) - ([views get:@"btninfocerrar"].Height));

}
@end

File: mobile_app/app_pedidos/Objects/src/com/jq/pedidos/designerscripts/LS_frminfoempresa.java
package com.jq.pedidos.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frminfoempresa{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("panelinfoback").vw.setLeft((int)(0d));
views.get("panelinfoback").vw.setWidth((int)((100d / 100 * width) - (0d)));
views.get("panelinfoback").vw.setTop((int)(0d));
views.get("panelinfoback").vw.setHeight((int)((100d / 100 * height) - (0d)));
views.get("panelinfo").vw.setLeft((int)((7d / 100 * width)));
views.get("panelinfo").vw.setHeight((int)((51d / 100 * height)));
views.get("panelinfo").vw.setTop((int)((50d / 100 * height) - (views.get("panelinfo").vw.getHeight() / 2)));
views.get("panelinfo").vw.setWidth((int)((100d / 100 * width)-(views.get("panelinfo").vw.getLeft())*2d));
views.get("lblinfoempresa").vw.setTop((int)((3d / 100 * height)));
views.get("lblinfoempresa").vw.setLeft((int)((5d / 100 * width)));
views.get("lblinfoempresa").vw.setHeight((int)((5d / 100 * height)));
views.get("lblinfoempresa").vw.setWidth((int)((views.get("panelinfo").vw.getWidth())-(views.get("lblinfoempresa").vw.getLeft())));
views.get("label1").vw.setLeft((int)((views.get("lblinfoempresa").vw.getLeft())));
views.get("label1").vw.setHeight((int)((5d / 100 * height)));
views.get("label1").vw.setWidth((int)((35d / 100 * width)));
views.get("label1").vw.setTop((int)((views.get("lblinfoempresa").vw.getTop() + views.get("lblinfoempresa").vw.getHeight())+(1d / 100 * height)));
views.get("lblinfonegocio").vw.setTop((int)((views.get("label1").vw.getTop())));
views.get("lblinfonegocio").vw.setLeft((int)((views.get("label1").vw.getLeft() + views.get("label1").vw.getWidth())));
views.get("lblinfonegocio").vw.setHeight((int)((views.get("label1").vw.getHeight())));
views.get("lblinfonegocio").vw.setWidth((int)((views.get("panelinfo").vw.getWidth())-(views.get("lblinfonegocio").vw.getLeft())));
views.get("label2").vw.setLeft((int)((views.get("label1").vw.getLeft())));
views.get("label2").vw.setWidth((int)((views.get("label1").vw.getWidth())));
views.get("label2").vw.setHeight((int)((views.get("label1").vw.getHeight())));
views.get("label2").vw.setTop((int)((views.get("label1").vw.getTop() + views.get("label1").vw.getHeight())+(1d / 100 * height)));
views.get("lblpedidominimo").vw.setTop((int)((views.get("label2").vw.getTop())));
views.get("lblpedidominimo").vw.setLeft((int)((views.get("label1").vw.getLeft() + views.get("label1").vw.getWidth())));
views.get("lblpedidominimo").vw.setHeight((int)((views.get("label1").vw.getHeight())));
views.get("lblpedidominimo").vw.setWidth((int)((views.get("lblinfonegocio").vw.getWidth())));
views.get("label3").vw.setLeft((int)((views.get("label1").vw.getLeft())));
views.get("label3").vw.setWidth((int)((views.get("label1").vw.getWidth())/1.5d));
views.get("label3").vw.setHeight((int)((views.get("label1").vw.getHeight())));
views.get("label3").vw.setTop((int)((views.get("label2").vw.getTop() + views.get("label2").vw.getHeight())+(1d / 100 * height)));
views.get("lbltelefono").vw.setTop((int)((views.get("label3").vw.getTop())));
views.get("lbltelefono").vw.setLeft((int)((views.get("label3").vw.getLeft() + views.get("label3").vw.getWidth())));
views.get("lbltelefono").vw.setHeight((int)((views.get("label1").vw.getHeight())));
views.get("lbltelefono").vw.setWidth((int)((views.get("lblinfonegocio").vw.getWidth())));
views.get("label4").vw.setLeft((int)((views.get("label1").vw.getLeft())));
views.get("label4").vw.setWidth((int)((views.get("label1").vw.getWidth())));
views.get("label4").vw.setHeight((int)((views.get("label1").vw.getHeight())));
views.get("label4").vw.setTop((int)((views.get("label3").vw.getTop() + views.get("label3").vw.getHeight())+(1d / 100 * height)));
views.get("lbldireccion").vw.setTop((int)((views.get("label4").vw.getTop() + views.get("label4").vw.getHeight())));
views.get("lbldireccion").vw.setLeft((int)((views.get("label4").vw.getLeft())));
views.get("lbldireccion").vw.setHeight((int)((10d / 100 * height)));
views.get("lbldireccion").vw.setWidth((int)((views.get("panelinfo").vw.getWidth())-(views.get("lbldireccion").vw.getLeft())*2d));
views.get("btninfocerrar").vw.setWidth((int)((25d / 100 * width)));
views.get("btninfocerrar").vw.setHeight((int)((7d / 100 * height)));
views.get("btninfocerrar").vw.setLeft((int)((views.get("panelinfo").vw.getWidth())-(2d / 100 * width) - (views.get("btninfocerrar").vw.getWidth())));
views.get("btninfocerrar").vw.setTop((int)((views.get("panelinfo").vw.getHeight())-(1d / 100 * height) - (views.get("btninfocerrar").vw.getHeight())));

}
}